scripts/new-customer-key.sh

```diff
@@ -27,6 +34,11 @@ case $subcmd in
 	# 3mdeb Dasharo Master Key
 	signing_kid=0D5F6F1DA800329EB7C597A2ABE1D0BC66278008
 	;;
+    pce)
+	real_name="PC Engines open-source firmware release ${version} signing key"
+	# 3mdeb Open Source Software Master Key <contact@3mdeb.com>
+	signing_kid=6058A306EF96BCBF56567B8C80693E028589B763
```

Comment:
Wrong one. This should be :

3mdeb Open Source Firmware Master Key <contact@3mdeb.com>
EA61CBADDA1C094A2540E596028A752764CB97EC

Reply:
@miczyg1 fixed